lib/logidze/meta.rb

```diff
@@ -0,0 +1,76 @@
+# frozen_string_literal: true
+
+require 'logidze/pg_helpers'
+
+module Logidze # :nodoc:
+  # Provide methods to attach meta information
+  module Meta
+    def with_meta(meta, &block)
+      MetaTransaction.wrap_with(meta, &block)
+    end
+
+    def with_responsible(responsible_id, &block)
+      return yield if responsible_id.nil?
+      meta = { Logidze::History::Version::RESPONSIBLE => responsible_id }
```

Comment:
Do we have a risk that users can inadvertently overwrite this key with their own meta? Yeah, I know that the usage of key `r` is unlikely, but hey, it's software development  :) Maybe `_r` or something would be safer?

Reply:
Yep, there are couple of options we could consider:

1. do nothing :)
2. use `_r` and hope that no one would override it
3. exclude `r` key (or whatever we end up with) from the meta hash silently
4. throw an exception when someone sets `r` meta key